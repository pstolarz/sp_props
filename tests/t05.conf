1;

2; scope 3 {} 4;

5 {
    1;
    scope 2 {}
    3;

}
5 {}
2;
