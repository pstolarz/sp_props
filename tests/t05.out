--- Del prop 1, own-scope: /, elm:0, flags:EXTEOL
2; scope 3; 4;

5 {
    1;
    scope 2;
    3;

}
5 {}
2;

--- Del prop 2, own-scope /, elm:ALL
1;

scope 3; 4;

5 {
    1;
    scope 2;
    3;

}
5 {}

--- Del scope:3, own-scope /, elm:LAST
1;

2; 4;

5 {
    1;
    scope 2;
    3;

}
5 {}
2;

--- Del prop 4, own-scope /, elm:0, flags:EXTEOL
1;

2; scope 3;
5 {
    1;
    scope 2;
    3;

}
5 {}
2;

--- Del prop 4, own-scope /, elm:LAST
1;

2; scope 3;

5 {
    1;
    scope 2;
    3;

}
5 {}
2;

--- Del scope :5, own-scope /, elm:LAST, flags:EXTEOL
1;

2; scope 3; 4;

5 {
    1;
    scope 2;
    3;

}
2;

--- Del scope :5, own-scope /, elm:ALL
1;

2; scope 3; 4;

2;

--- Del prop 1, own-scope /:5, elm:0
1;

2; scope 3; 4;

5 {
    scope 2;
    3;

}
5 {}
2;

--- Del prop 3, own-scope /:5, elm:0, flags:EXTEOL
1;

2; scope 3; 4;

5 {
    1;
    scope 2;
}
5 {}
2;

--- Del prop 1, parsing scope /:5, elm:0
    scope 2;
    3;
--- Del prop 3, parsing scope /:5, elm:0, flags:EXTEOL
1;
    scope 2;

--- Del absent prop 3, own-scope /
1;

2; scope 3; 4;

5 {
    1;
    scope 2;
    3;

}
5 {}
2;

--- Del prop 1 in absent own-scope /:1
1;

2; scope 3; 4;

5 {
    1;
    scope 2;
    3;

}
5 {}
2;
